src/Provider/AbstractProvider.php

```diff
@@ -356,12 +356,7 @@ protected function fetchProviderData($url, array $headers = [])
         try {
             $client = $this->getHttpClient();
             $client->setBaseUrl($url);
```

Comment:
Couldn't we also drop the `setBaseUrl` call here and use `get($url, $headers)` instead?